src/helper/client.cpp

```diff
@@ -221,7 +263,7 @@ bool client::handle_command(network::config_sync::request & /* command */)
 
     SPDLOG_DEBUG("received command config_sync");
 
-    if (service_->get_service_config()->is_asm_enabled()) {
+    if (compute_extension_status()) {
```

Comment:
Don't refer to `extension` in the helper, instead refer to `client`.

Reply:
ok

Reply:
Done